python/pysmurf/client/base/smurf_control.py

```diff
@@ -446,8 +446,9 @@ def setup(self, write_log=True, **kwargs):
         # Turn on stream enable for all bands
         self.set_stream_enable(1, write_log=write_log)
 
-        # self.set_smurf_to_gcp_clear(1, write_log=write_log, wait_after=1)
-        # self.set_smurf_to_gcp_clear(0, write_log=write_log)
+        # Set mask file to empty array and payload size
+        self.set_payload_size(2048)
```

Comment:
Any way to not hard code?

Reply:
Pushed opt arg.

Reply:
https://github.com/slaclab/pysmurf/pull/272/commits/429aa2e5ca01c2953ca5d7f19a8fa431b75268c7